src/utils.ts

```diff
@@ -49,12 +50,21 @@ export function formatErrors(
     loaderOptions: LoaderOptions,
     colors: Chalk,
     compiler: typeof typescript,
-    merge?: { file?: string; module?: WebpackModule }
+    merge: { file?: string; module?: WebpackModule }
 ): WebpackError[] {
 
     return diagnostics
         ? diagnostics
             .filter(diagnostic => loaderOptions.ignoreDiagnostics.indexOf(diagnostic.code) === -1)
```

Comment:
Could we merge the 2 filter statements in `utils` please? I'd only like to iterate the collection a single time if possible.

Reply:
sure will do